docker-registry-sync

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+usage() {
+  echo "$0:
+
+Description: Syncs images from a public docker registry to a private registry. Use this to populate private registries in a closed off environment.
+Must be run from a linux host capable of running docker commands which has access both to the internet and the private registry.
+
```

Comment:
Maybe include a comment about the format of the "image-file" - i.e.: one entry per line, etc.